### ISSUE TITLE
Fed Reserve FOMC views on interest rates staying high at least into 2024

### ISSUE THREAD

**Issue by dawu76** [dawu76/investment-research-notes#91]
As her 40-year central banking career comes to a close, Kansas City Federal Reserve President Esther George is advising her colleagues to stay tough in their efforts to stamp out runaway inflation.

George said Thursday that she thinks the Fed should raise its benchmark borrowing rate above 5% and keep it there until there are substantial signs that prices are stabilizing.

"Holding that until we get evidence that inflation is actually coming down is really the message we're trying to put out there," she told CNBC's Steve Liesman during a "Squawk Box" interview. "I'll be over 5% and I see staying there for some time, again until we get the signal that inflation is really convincingly starting to fall back toward our 2% goal."

At the December Fed meeting, the rate-setting Federal Open Market Committee voted to raise the fed funds rate half a percentage point to a range of 4.25%-4.5%.

Meeting minutes released Wednesday indicated that members see no chance of any rate cuts in 2023, and they expressed concern over whether the public mistakenly might view the step down in rate hikes, from a string of four straight three-quarter point moves, as a softening in policy.

Asked whether her view is that the funds rate should hold above 5% into 2024, George replied, "It is for me." That statement comes a day after Minneapolis Fed President Neel Kashkari wrote that he thinks the funds rate should rise to 5.4% and could go even higher if inflation doesn't come down.

In previous comments, George has said the tighter monetary policy is expected to tamp down demand and slow economy, possibly enough to create a recession. She said in her remarks to CNBC that she doesn't see that as inevitable, but rather as a possibility.

"I'm not forecasting a recession," she said. "But I'm quite realistic that when you see below-trend growth and the idea that our instrument is going to work on demand, bringing that down, it doesn't leave a lot of margin there. Any shock could come, any risk to the outlook could send the economy in that direction. So it's not my forecast, but I do understand that bringing demand down creates that sort of possibility."

... FOMC minutes: Officials further cautioned that the public shouldn't read too much into the rate-setting Federal Open Market Committee's move to step down the pace of increases.

"A number of participants emphasized that it would be important to clearly communicate that a slowing in the pace of rate increases was not an indication of any weakening of the Committee's resolve to achieve its price-stability goal or a judgment that inflation was already on a persistent downward path," the minutes said.

The minutes reflected those sentiments, noting that no FOMC members expect rate cuts in 2023, despite market pricing.

Markets currently are pricing in the likelihood of rate increases totaling 0.5-0.75 percentage point before pausing to evaluate the impact the hikes are having on the economy. Traders expect the central bank to approve a quarter-point increase at the next meeting, which concludes Feb. 1, according to CME Group data.

Current pricing also indicates the possibility of a small reduction in rates by the end of the year, with the funds rate landing around a range of 4.5%-4.75%. Fed officials, however, have expressed doubt repeatedly about any loosening of policy in 2023.

Along with the rate hikes, the Fed has been reducing the size of its balance sheet by allowing up to $95 billion in proceeds from maturing securities to roll off each month rather than be reinvested. In a program started in early June, the Fed has seen its balance sheet contract by $364 billion to $8.6 trillion.

While some of the recent inflation metrics have shown progress, the labor market, a critical target of the rate increases, has been resilient. Nonfarm payroll growth has exceeded expectations for most of the past year, and data earlier Wednesday showed that the number of job openings is still nearly twice the pool of available workers.

The Fed's preferred inflation gauge, the personal consumption expenditures price index less food and energy, was at 4.7% annually in November, down from its 5.4% peak in February 2022 but still well above the Fed's 2% target.

Economists, meanwhile, largely expect the U.S. to enter a recession in the coming months, the result of the Fed's tightening and an economy dealing with inflation still running near 40-year highs. However, fourth-quarter GDP for 2022 is tracking at a solid 3.9% rate, easily the best of a year that started out with consecutive negative readings, according to the Atlanta Fed.

Minneapolis Fed President Neel Kashkari said Wednesday, in a post for the district's website, that he sees the funds rate rising to 5.4% and possibly higher if inflation doesn't trend down.

... Kashkari: Imagine a rainstorm hits, and the price surges from $20 to $90. But imagine, in my analogy, the ride-sharing company owns all the cars, and the drivers are its employees. When the storm hits, the company notifies all their drivers that they will pay time and a half. So, all the drivers come out to drive. The company doesn’t raise wages even more because they are car constrained: All their cars are deployed.

My story is of course too simple — just a demand surge, rather than both increased demand and disrupted supply — but the resulting characteristics resemble the economy we have been experiencing: Prices soar. Corporate profits climb. Income for drivers climbs, but not as much as prices. Real wages actually fall. Even though worker incomes are up, labor’s share of income is down. Labor markets are tight, but capital is the constraint on supply. Inflation has soared, but it soared because of supply/demand dynamics — what I will call “surge pricing inflation.” Economists would say the market hit the vertical part of the supply curve.

The inflation in this example is not driven by the two primary sources that traditional Phillips-curve models used by policymakers, researchers, and investors consider: (1) labor market effects via unemployment gaps, and (2) changes to long-run inflation expectations.

In these workhorse models, it is very difficult to generate high inflation: Either we need to assume a very tight labor market combined with nonlinear effects, or we must assume an unanchoring of inflation expectations. That’s it. From what I can tell, our models seem ill-equipped to handle a fundamentally different source of inflation, specifically, in this case, surge pricing inflation.

This is a challenge for economists inside and outside the Fed: Can we develop frameworks and tools to analyze and potentially forecast inflation outside of labor market and expectations channels? Specifically surge pricing inflation, but potentially others as well?

If we can deepen our analytical capabilities surrounding other sources and channels of inflation, then we might be able to incorporate whatever lessons we learn into our policy framework going forward.

One may ask why tightening monetary policy is the right response to what I described as surge pricing inflation. Unfortunately, the initial surge in inflation is leading to broader inflationary pressures that the Federal Reserve must control. For example, nominal wage growth has grown to 5 percent or more, which is inconsistent with our 2 percent inflation target given recent trend productivity growth. Monetary policy is the appropriate tool to bring the labor market back into balance.

In my view, however, it will be appropriate to continue to raise rates at least at the next few meetings until we are confident inflation has peaked.

Once we reach that point, then the second step of our inflation fighting process, as I see it, will be pausing to let the tightening we have already done work its way through the economy. I have us pausing at 5.4 percent (see figure), but wherever that end point is, we won’t immediately know if it is high enough to bring inflation back down to 2 percent in a reasonable period of time. Once we see the full effects of the tightened policy, we can then assess whether we need to go higher or simply remain at that peak level for longer. To be clear, in this phase any sign of slow progress that keeps inflation elevated for longer will warrant, in my view, taking the policy rate potentially much higher.

The third step, as I see it, is to consider cutting rates only once we are convinced inflation is well on its way back down to 2 percent. Given the experience of the 1970s, the mistake the FOMC must avoid is to cut rates prematurely and then have inflation flare back up again.
